inventory/environ.py

```diff
@@ -203,9 +203,10 @@ def loadDefaultSplunkVariables():
         loaded_yaml = yaml.load(yaml_file)
     
     ### Load the defaults for the environment
```

Comment:
Oy, am I missing something or is this:
```
    vars_scope["ansible_pre_tasks"] = os.environ.get("SPLUNK_ANSIBLE_PRE_TASKS") or getValueFromDEFAULTS("ansible_pre_tasks")
    vars_scope["ansible_post_tasks"] = os.environ.get("SPLUNK_ANSIBLE_POST_TASKS") or getValueFromDEFAULTS("ansible_post_tasks")
```
missing from the final commit.

Reply:
think I lost it during the merge, good catch.